railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -0,0 +1,50 @@
+# Make sure it matches the Ruby version in .ruby-version and Gemfile
```

Comment:
It could be useful to mention [`--build-arg`](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg) to help with debugging out-of-sync Ruby versions.